x-pack/index.js

```diff
@@ -28,6 +28,7 @@ import { kueryAutocomplete } from './plugins/kuery_autocomplete';
 import { canvas } from './plugins/canvas';
 import { infra } from './plugins/infra';
 import { rollup } from './plugins/rollup';
+import { remoteClustersManagement } from './plugins/remote_clusters_management';
```

Comment:
Open to feedback on this naming. I am calling it `remoteClustersManagement` to avoid user confusion between a licensed Kibana feature and remote cluster support in ES (which is OSS).

Reply:
This makes sense to me. If you wanted to shorten it I think we could also go with `remoteClustersApp` or `remoteClustersPlugin`, but I don't feel strongly about it.